www/src/pages/components/carousel/ios/index.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: CarouselLayout
```

Comment:
```suggestion
title: CarouselLayout
title: Carousel
```

The page has to have the same name as the component on the other platforms otherwise the documentation isn't able to group it together:

https://deploy-preview-707--thumbprint.netlify.app/components/carousel/ios/

Would it be 👍 to name the page title "Carousel" even if the concept within the codebase is "CarouselLayout"? FWIW the search functionality would pick up both terms.